phpcs.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<ruleset name="temporary">
+	<description>Temporary coding standard pending project maintainer review.</description>
+	<arg name="colors" />
+	<arg name="tab-width" value="2" />
+	<!-- Enforce short array syntax for PHP 5.5+ module -->
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax" />
+	<!-- Custom line length validation -->
+	<rule ref="Generic.Files.LineLength">
+		<properties>
+			<property name="lineLimit" value="80" />
+			<property name="absoluteLineLimit" value="100" />
+		</properties>
+	</rule>
+	<rule ref="Generic.WhiteSpace.ScopeIndent">
+		<properties>
+			<property name="indent" value="2" />
+			<property name="tabIndent" value="true" />
```

Comment:
@arswright able to do spaces?

Reply:
Oh, right - you'll want to remove the `tabIndent` directive :)